### PR TITLE
Adding application icon

### PR DESCRIPTION
**Pull request by armandojaga**

### CODE REVIEW

**Comment by Bernatmago** [comment]
Who would have said that adding an icon needed so much code LOL.

**Review by Bernatmago — APPROVED** [review]

File: Source/Hachiko.rc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Icon
+//
+
+// Icon with lowest ID value placed first to ensure application icon
+// remains consistent on all systems.
+IDI_ICON1               ICON                    "C:\\CODE\\UPC\\Hachiko-Engine\\hachiko.ico"
```
Comment (bernatgb) [inline Source/Hachiko.rc:55]:
```suggestion
IDI_ICON1               ICON                    "..\\hachiko.ico"
```

Comment (bernatgb) [reply Source/Hachiko.rc:55]: The path should be changed.

**Review by bernatgb — APPROVED** [review]
Hachiko looking fancy!!

**Pull request merged**